clients/filesystem-hadoop3/build.gradle.kts

```diff
@@ -71,6 +75,11 @@ tasks.build {
   dependsOn("javadoc")
 }
 
+tasks.compileJava {
```

Comment:
Why do we need this?

Reply:
This module depends on `catalog-hadoop`, however, due to the fact that the output directory of task `:catalogs:catalog-hadoop:jar` and `:catalogs:catalog-hadoop:runtimeJars` are the same (lib/build/), so we need to add these two dependency or gradle compile will fail, the same goes for others module 

```gradle
tasks.test {
  val skipITs = project.hasProperty("skipITs")
  if (skipITs) {
    exclude("**/integration/test/**")
  } else {
    dependsOn(":catalogs:catalog-hadoop:jar", ":catalogs:catalog-hadoop:runtimeJars")
    dependsOn(":catalogs:catalog-hive:jar", ":catalogs:catalog-hive:runtimeJars")
    dependsOn(":catalogs:catalog-kafka:jar", ":catalogs:catalog-kafka:runtimeJars")
  }
}
```